recipes/ena-webin-cli/meta.yaml

```
@@ -0,0 +1,27 @@
+{% set version="3.7.0" %}
+
+about:
+  home: https://github.com/enasequence/webin-cli
+  license: Apache-2.0 License
+  summary: Data submissions to ENA can be made using the Webin command line submission interface (Webin-CLI).
+
+package:
+  name: ena-webin-cli
+  version: {{ version }}
+
+build:
+  noarch: generic
+  number: 0
+
+source:
+    url: https://github.com/enasequence/webin-cli/releases/download/v3.7.0/webin-cli-3.7.0.jar
```

```suggestion
    url: https://github.com/enasequence/webin-cli/releases/download/v{{ version }}/webin-cli-{{ version }}.jar
```

I think this is needed